Resources/Prototypes/Entities/Mobs/NPCs/simplemob.yml

```
@@ -163,7 +163,21 @@
   - type: InputMover
   - type: MobMover
   - type: Hunger
+    thresholds: # only animals and rats are derived from this prototype so let's override it here and in rats' proto
+      Overfed: 100
+      Okay: 50
+      Peckish: 25
+      Starving: 10
+      Dead: 0
+    baseDecayRate: 0,00925925925926 # it is okay for animals to eat and drink less than humans, but more frequently
```

```suggestion
    baseDecayRate: 0.00925925925926 # it is okay for animals to eat and drink less than humans, but more frequently
```

noooot sure how the linter didnt catch this